python/src/nanoarrow/_lib.pyx

```diff
@@ -947,88 +926,28 @@ cdef class BufferView:
     def __releasebuffer__(self, Py_buffer *buffer):
         pass
 
+    def __repr__(self):
+        return _lib_utils.buffer_view_repr(self)
```

Comment:
It might be nice to include a name here as well for the standalone repr (the util function only gives you the content, which is useful for including it into another repr).
Something like 

```suggestion
        return f"nanoarrow.c_lib.BufferView {_lib_utils.buffer_view_repr(self)[1:]}"
```

(the slicing is because it already starts with a `<` (that could also be changed in the util function)